src/components/Match/Vision/VisionLog.jsx

```diff
@@ -58,7 +60,8 @@ const generateData = match => (log) => {
 
 const VisionLog = ({ match, wards }) => (
   <div className={styles.visionLog}>
```

Comment:
remove this class

![image](https://cloud.githubusercontent.com/assets/20866892/23668784/c26d3706-036a-11e7-8059-3f6d6fe138bf.png)